src/heretic/r_plane.c

```diff
@@ -587,18 +590,18 @@ void R_DrawPlanes(void)
             case 28:
             case 29:           // Scroll_North
                 xsmoothscrolloffset = 0;
-                ysmoothscrolloffset = 0;
+                ysmoothscrolloffset = (crispy->scrolleranims) ? FLATSCROLL(pl->special - 25) : 0;
                 ds_source = tempSource;
                 break;
             case 20:
             case 21:
             case 22:
             case 23:
             case 24:           // Scroll_East
-                xsmoothscrolloffset = -(interpfactor << (pl->special - 20));
+                // vanilla Heretic always animates eastward scrollers
+                xsmoothscrolloffset = -(FLATSCROLL(pl->special - 20));
```

Comment:
No need for an extra pair of parentheses, the macro is already embraced in a pair.

Reply:
The entire macro is not enclosed in parenthesis, `(interpfactor << (X))` is separately-bracketed from the rest. I can enclose enclose the whole thing if you prefer.